zellij-utils/assets/prost/api.action.rs

```diff
@@ -587,6 +591,33 @@ impl ActionName {
             "BreakPaneLeft" => Some(Self::BreakPaneLeft),
             "RenameSession" => Some(Self::RenameSession),
             "LaunchPlugin" => Some(Self::LaunchPlugin),
+            "MoveTab" => Some(Self::MoveTab),
+            _ => None,
+        }
+    }
+}
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, ::prost::Enumeration)]
+#[repr(i32)]
+pub enum MoveTabDirection {
```

Comment:
Can you move this to line 329? I think it would be better for visibility if Enum are placed close to each other.

Reply:
Yep, done :+1: